Taskfile.yml

```diff
@@ -298,7 +298,7 @@ tasks:
     cmds:
       - >
         docker-compose exec mongodb mongosh mongodb://host.docker.internal:27017/test?heartbeatFrequencyMS=300000
-        --verbose --eval 'disableTelemetry()' --shell
+        --verbose --eval 'disableTelemetry()' --shell /mongosh/test.js
```

Comment:
This is nice, I had to [look up](https://www.mongodb.com/docs/mongodb-shell/reference/options/#std-option-mongosh.--shell) :see_no_evil: